website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Terraform Registry FAQ
+description: Troubleshooting common Terraform Registry problems
+---
+
+## Common publishing issues
+
+### How does the Registry identify the latest version of my provider, module, or policy library?
+
+Each version of an integration published in the Terraform Registry has a version identifier, which is a string like `1.2.3`. You create the version identifier when you create a tag (for modules and policy libraries) or a release (for providers) in the GitHub repository where the integration's code lives.
+
+The Registry requires that these version identifiers are [SemVer compliant](https://semver.org/). Versions are sorted using these identifiers, following [standard SemVer rules for precedence](https://semver.org/#spec-item-11). The version identifier with the greatest precedence is always listed as latest by the Terraform Registry.
```

Comment:
The Terraform CLI won't pick a prerelease version. It would make sense to call that out here.